src/PKSim.Infrastructure/ProjectConverter/v10/Converter9To10.cs

```diff
@@ -152,31 +153,60 @@ private void convertMolecule(IndividualMolecule moleculeToConvert, Individual in
          if (newMolecule == null)
             return;
 
+         newMolecule.Ontogeny = moleculeToConvert.Ontogeny;
+
          var allExpressionParameters = individual.AllExpressionParametersFor(newMolecule);
-         moleculeToConvert.GetAllChildren<MoleculeExpressionContainer>().Each(x =>
-         {
-            convertParameter(x.RelativeExpressionParameter, allExpressionParameters[x.Name]);
-         });
+         moleculeToConvert.GetAllChildren<MoleculeExpressionContainer>().Each(x => { convertParameter(x.RelativeExpressionParameter, allExpressionParameters[x.Name]); });
```

Comment:
Tja....this was a mistake. The old moelcule is removed so everything needs to be copied again. Forgot about this. Tests everywhere

Reply:
looks good